src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -1,10 +1,25 @@
 import React, { Children } from 'react';
+import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
 
 import Button from '../Button';
 import { childrenPropType } from '../../util/shared-prop-types';
+import { isEqual } from '../../util/fp';
 
-const listStyles = ({ theme }) => css`
+const LEFT = 'left';
+const RIGHT = 'right';
+const ALIGMENT_TYPES = [LEFT, RIGHT];
+
+const isRight = isEqual(RIGHT);
```

Comment:
Love it, but I'm not sure if this is taking it too far. I'd be very fine with doing `align === RIGHT` in the ternary condition.

Reply:
I'm gonna rewrite this way then.